actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -480,6 +487,8 @@ def []=(name, options)
           options = { value: options }
         end
 
+        preprocess_options(options)
```

Comment:
No, I meant `respond_to?(:from_now)` in lieu of `is_a?` in my proposed `expiry_options`.